runelite-client/src/test/java/net/runelite/client/util/ColorUtilTest.java

```diff
@@ -44,6 +44,12 @@
 		put(Color.WHITE, "ffffff");
 	}};
 
+	private static final Map<Color, String> COLOR_ALPHA_HEXSTRING_MAP = new HashMap<Color, String>()
```

Comment:
use `ImmutableMap.of`

Reply:
Should I update the `COLOR_HEXSTRING_MAP` above too while I'm at it?